atomicapp/nulecule/lib.py

```diff
@@ -64,46 +63,21 @@ def load_config(self, config, ask=False, skip_asking=False):
             None
         """
         for param in self.params:
-            value = config.get(self.namespace, {}).get(param[NAME_KEY]) or \
-                config.get(GLOBAL_CONF, {}).get(param[NAME_KEY])
+            value = config.get(param[NAME_KEY], scope=self.namespace)
```

Comment:
Created issue for this at: https://github.com/projectatomic/atomicapp/issues/780